src/main/webapp/app/exercises/programming/assess/repo-export/programming-assessment-repo-export-dialog.component.ts

```diff
@@ -73,6 +75,7 @@ export class ProgrammingAssessmentRepoExportDialogComponent implements OnInit {
                     this.isLoading = false;
                 });
         } else {
+            this.isAtLeastInstructor = this.selectedProgrammingExercises.filter((exercise) => !exercise.isAtLeastInstructor).length === 0;
```

Comment:
I think this could be re-written like so? `!this.selectedProgrammingExercises.some((exercise) => !exercise.isAtLeastInstructor)`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
Good point 👍 I added this with the last commit https://github.com/ls1intum/Artemis/pull/5731/commits/e504916f5d579604adaa0b97531e564c5e8d9cc4